db/seeds.rb

```diff
@@ -52,52 +52,46 @@ def random_record_for_org(org, klass)
   org.items.where(value_in_cents: 0).limit(10).update_all(value_in_cents: 100)
 end
 
-# super admin
-user = User.create email: 'superadmin@example.com', password: 'password', password_confirmation: 'password', organization_admin: false, super_admin: true
-
-# org admins
-user = User.create email: 'org_admin1@example.com', password: 'password', password_confirmation: 'password', organization: pdx_org, organization_admin: true
-user2 = User.create email: 'org_admin2@example.com', password: 'password', password_confirmation: 'password', organization: sf_org, organization_admin: true
-
-# regular users
-User.create email: 'user_1@example.com', password: 'password', password_confirmation: 'password', organization: pdx_org, organization_admin: false
-User.create email: 'user_2@example.com', password: 'password', password_confirmation: 'password', organization: sf_org, organization_admin: false
-
-# test users
-User.create email: 'test@example.com', password: 'password', password_confirmation: 'password', organization: pdx_org, organization_admin: false, super_admin: true
-User.create email: 'test2@example.com', password: 'password', password_confirmation: 'password', organization: sf_org, organization_admin: true
-
-DonationSite.find_or_create_by!(name: "Pawnee Hardware") do |location|
-  location.address = "1234 SE Some Ave., Pawnee, OR 12345"
-  location.organization = pdx_org
-end
-DonationSite.find_or_create_by!(name: "Parks Department") do |location|
-  location.address = "2345 NE Some St., Pawnee, OR 12345"
-  location.organization = pdx_org
-end
-DonationSite.find_or_create_by!(name: "Waffle House") do |location|
-  location.address = "3456 Some Bay., Pawnee, OR 12345"
-  location.organization = pdx_org
-end
-DonationSite.find_or_create_by!(name: "Eagleton Country Club") do |location|
-  location.address = "4567 Some Blvd., Eagleton, OR 12345"
-  location.organization = pdx_org
+[
+  { email: 'superadmin@example.com', organization_admin: false, super_admin: true },
+  { email: 'org_admin1@example.com', organization_admin: true, organization: pdx_org },
+  { email: 'org_admin2@example.com', organization_admin: true, organization: sf_org },
+  { email: 'user_1@example.com', organization_admin: false, organization: pdx_org },
```

Comment:
It would be nice if all records in a given array were vertically aligned, so that scanning a given field in multiple records would be easier. For example, this:
```
{ email: 'org_admin2@example.com', organization_admin: true, organization: sf_org },
{ email: 'user_1@example.com', organization_admin: false, organization: pdx_org },
```

changed to this:

```
{ email: 'org_admin2@example.com', organization_admin: true,  organization: sf_org },
{ email: 'user_1@example.com',     organization_admin: false, organization: pdx_org },
```